migrations/drop-audit-doc-index.js

```diff
@@ -22,7 +22,7 @@ var needsUpdate = function(row) {
 var createNewDocs = function(oldDocs, callback) {
   var newDocs = oldDocs.map(function(doc) {
     return {
-      _id: doc.record_id,
+      _id: doc.record_id + '-audit',
```

Comment:
Is this better structured as a prefix? Not a strong preference, but IIRC CouchDB uses prefixes as a convention and they probably have some highly-theoretical future benefits wrt indexing.